commands/random.js

```diff
@@ -0,0 +1,136 @@
+const format = require('../format')
+const os = require('os')
+
+module.exports.init = controller => {
+    controller.hears(['thanks', 'thank you'], 'direct_message,direct_mention,mention', (bot, message) => {
+        controller.storage.users.get(message.user, (err, user) => {
+            if (user && user.name) {
+                return bot.reply(message, 'You are most welcome, ' + user.name + '!!')
+            }
+
+            bot.reply(message, 'Anytime!')
+        })
+    })
+
+    controller.hears(['you around'], 'direct_message,direct_mention,mention', (bot, message) => {
+        controller.storage.users.get(message.user, (err, user) => {
+            if (user && user.name) {
+                return bot.reply(message, 'I\'m here for you, ' + user.name + '!!')
```

Comment:
You could use javascript string literals like this:
```
return bot.reply(message, `I'm here for you, ${user.name}!!`)
```

Reply:
random.js is left over from the original boilerplate. I haven't cleaned it up yet. I'd like to build a simpler interface to be able to add neat responses without having so much code surrounding it.